pkg/controllers/v1alpha1/dataload/dataload_controller.go

```diff
@@ -154,6 +136,18 @@ func (r *DataLoadReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			"Bounded accelerate runtime not supported")
 	}
 
+	if err != nil {
+		if utils.IgnoreNotFound(err) == nil {
+			ctx.Log.V(1).Info("The runtime is not found", "runtime", ctx.NamespacedName)
+			return ctrl.Result{}, nil
+		} else {
+			ctx.Log.Error(err, "Failed to get the ddc runtime")
+			return utils.RequeueIfError(errors.Wrap(err, "Unable to get ddc runtime"))
+		}
+	}
```

Comment:
suggest this error processing logic :
```go
return ctrl.Result{},errors.Wrapf(utils.IgnoreNotFound(err),"failed to get ddc runtime %s",ctx.NamespacedName)
```
`utils.RequeueIfError` there is no need for encapsulation, and it is not very readable